translation-cli/translation_cron.php

```diff
@@ -68,6 +68,55 @@ final class TranslationCron
 	 */
 	private $verbose = false;
 
+	/**
+	 * The list of J4 language packs to mark them as comptible
+	 *
+	 * @var array
+	 */
+	private $j4LanguagePack = array(
+		'ar-AA',
```

Comment:
wouldn't this be better alpha sorted

Reply:
It is sorted as the downloads sites showed them to me.

Reply:
they are alpha sorted by real name.

think about it. if hebrew is added where will you put it in the list. 
if welsh is removed how will you find it in this list.

keeping the sort order based on the download site will mean you need to build the entire list whenever there is a change

Reply:
That is a good point, will resort it later.